www/js/diary/list/TimelineScrollList.tsx

```diff
@@ -56,11 +57,22 @@ const TimelineScrollList = ({ listEntries }: Props) => {
     </LoadMoreButton>
   );
 
+  const pipelineEndDate = pipelineRange && DateTime.fromSeconds(pipelineRange.end_ts).toISODate();
   const noTravelBanner = (
     <Banner visible={true} icon={({ size }) => <Icon source="alert-circle" size={size} />}>
       <View style={{ width: '100%' }}>
         <Text variant="titleMedium">{t('diary.no-travel')}</Text>
         <Text variant="bodySmall">{t('diary.no-travel-hint')}</Text>
+        {queriedDateRange?.[0] && pipelineEndDate && queriedDateRange?.[0] > pipelineEndDate && (
+          <Button
+            style={{ marginEnd: 'auto' }}
+            labelStyle={{ marginHorizontal: 0 }}
+            onPress={() =>
+              loadDateRange([isoDateWithOffset(pipelineEndDate, -6), pipelineEndDate])
+            }>
+            {t('diary.jump-to-last-processed-week')}
```

Comment:
The main reason for this decision is that the old implementation (which loaded `[pipelineEnd - 1 week, TODAY]`) would cause more than 7 days to be loaded initially if pipelineEnd != TODAY

In the old Dashboard tab, it didn't matter how many days were loaded because only the last 14 days of data were used. Now we have cumulative totals for footprint, which is the first thing the user will see when they open the Dashboard in the default user flow. So it becomes important for the initial load to always be a consistent number of days loaded.
Otherwise, the user may not be able to compare this value over time (e.g. "Yesterday when I opened my Dashboard and there were 16 days, but today when I opened it there were only 14 days")

A secondary reason is that for users with a stuck pipeline, the old implementation was nearly unusable.
The stuck pipeline is not an expected state, but we should still be able to collect labels on the draft trips. When I open the app right now, more than 300 trips show up, the UI is laggy, and it is difficult to label anything. After this change, the initial load is only the last week and I think it will be more manageable